src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,118 @@
+import { Icon } from "@/components";
+import { IconName } from "@/components/Icon/types";
+import { useState } from "react";
+import styled from "styled-components";
+
+type AlertSize = "small" | "medium";
+type AlertState = "neutral" | "success" | "warning" | "danger" | "info";
+export type AlertProps = {
+  state?: AlertState;
```

Comment:
maybe `type` and make it required?

Reply:
I am following the names as used in figma so that it's easier to work together with the design team